### PR TITLE
Feature/86cw10vv5 artist profile external view

### PR DESCRIPTION
**Pull request by NikkiShao**
Artist's profile from an external view
- services tabs
image
- service area tab (this is not completely done, the service area radius is not done but is NOT part of this ticket)
- some areas might show up in the sea, due to fake location information not being found in mapbox
image

### CODE REVIEW

File: behind-the-veil-siteroot/imports/api/publications/services.js
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/api/publications/services.js:1]: This is an old change but I realised it needs a fix but L8 needs to import { ServiceCollection } from "../collections/services"; cos ur in the current directory anyway

**Review by nethbtju — DISMISSED** [review]
Helper function has discussed

**Comment by NikkiShao** [comment]
still working on helper function (do not merge yet)

File: behind-the-veil-siteroot/imports/api/publications/services.js
Comment (NikkiShao) [reply behind-the-veil-siteroot/imports/api/publications/services.js:1]: done

**Review by nethbtju — DISMISSED** [review]
Fix merge conflict

**Review by nethbtju — APPROVED** [review]

**Pull request merged**